<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipModif</class>
 <widget class="QDialog" name="EquipModif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier un équipement</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/ressources.qrc">
    <normaloff>:/prog-data/img/icone.png</normaloff>:/prog-data/img/icone.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>20</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="malusGroup">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>113</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Malus</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="malusLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="couLabel">
       <property name="text">
        <string>COU :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="cou"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="intelLabel">
       <property name="text">
        <string>INT :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="intel"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="chaLabel">
       <property name="text">
        <string>CHA :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="cha"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="adLabel">
       <property name="text">
        <string>AD :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="ad"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="foLabel">
       <property name="text">
        <string>FO :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="fo"/>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="atLabel">
       <property name="text">
        <string>AT :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="at"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="prdLabel">
       <property name="text">
        <string>PRD :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="prd"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>21</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="bonusGroup">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>113</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Bonus</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="bonusLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="COULabel">
       <property name="text">
        <string>COU :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="COU"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="INTELLabel">
       <property name="text">
        <string>INT :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="INTEL"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="CHALabel">
       <property name="text">
        <string>CHA :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="CHA"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ADLabel">
       <property name="text">
        <string>AD :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="AD"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="FOLabel">
       <property name="text">
        <string>FO :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="FO"/>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ATLabel">
       <property name="text">
        <string>AT :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="AT"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="PRDLabel">
       <property name="text">
        <string>PRD :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="PRD"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="layoutNom">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nomLabel">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nom">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="layoutDivers">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="ruptLabel">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Rupture max (1 à x) :&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(si incassable, laissez à 0)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="rupt">
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="prLabel">
      <property name="text">
       <string>Protection :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="pr"/>
    </item>
    <item row="3" column="0" colspan="2">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="deLabel">
      <property name="text">
       <string>Nombre de dés d'impact :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="de"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="PILabel">
      <property name="text">
       <string>Points d'impact :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="PI">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="nbLabel">
      <property name="text">
       <string>Nombre de flèches :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="nb"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="vider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;&amp;Enlever&quot; cet équipement</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>243</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="typeLabel">
      <property name="text">
       <string>Type d'arme :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="type">
      <item>
       <property name="text">
        <string>A mains nues</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arme contandante</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Arme tranchante</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Projectile</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nom</tabstop>
  <tabstop>COU</tabstop>
  <tabstop>INTEL</tabstop>
  <tabstop>CHA</tabstop>
  <tabstop>AD</tabstop>
  <tabstop>FO</tabstop>
  <tabstop>AT</tabstop>
  <tabstop>PRD</tabstop>
  <tabstop>cou</tabstop>
  <tabstop>intel</tabstop>
  <tabstop>cha</tabstop>
  <tabstop>ad</tabstop>
  <tabstop>fo</tabstop>
  <tabstop>at</tabstop>
  <tabstop>prd</tabstop>
  <tabstop>rupt</tabstop>
  <tabstop>pr</tabstop>
  <tabstop>de</tabstop>
  <tabstop>PI</tabstop>
  <tabstop>nb</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>vider</tabstop>
 </tabstops>
 <resources>
  <include location="../res/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EquipModif</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>10</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EquipModif</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
